sdks/python/apache_beam/io/avroio.py

```diff
@@ -324,3 +332,9 @@ def open(self, temp_path):
 
   def write_record(self, writer, value):
     writer.append(value)
+
+  def display_data(self):
+    res = super(self.__class__, self).display_data()
+    res['codec'] = str(self._codec)
+    res['schema'] = str(self._schema)
```

Comment:
@chamikaramj The avro schema is a large, convoluted string. I'd like your input on whether you think it should be included in the display data of an avro sink? : )